src/dbnode/persist/fs/files.go

```diff
@@ -115,12 +115,18 @@ func (f FileSetFile) IsZero() bool {
 	return len(f.AbsoluteFilepaths) == 0
 }
 
-// HasCheckpointFile returns a bool indicating whether the given set of
+// HasCompleteCheckpointFile returns a bool indicating whether the given set of
 // fileset files has a checkpoint file.
-func (f FileSetFile) HasCheckpointFile() bool {
+func (f FileSetFile) HasCompleteCheckpointFile() bool {
 	for _, fileName := range f.AbsoluteFilepaths {
 		if strings.Contains(fileName, checkpointFileSuffix) {
```

Comment:
Probably don't need this line anymore since `CompleteCheckpointFileExists` does this already.

Reply:
True but that will cause an "Invariant error" to be allocated which I'd like to avoid in the case that we're scanning a lot of files.